crates/bevy_winit/src/winit_config.rs

```diff
@@ -28,7 +28,7 @@ pub struct WinitSettings {
     /// The runner will panic if this is set to `true` on other platforms.
     ///
     /// [`run`]: https://docs.rs/winit/latest/winit/event_loop/struct.EventLoop.html#method.run
-    /// [`run_return`]: https://docs.rs/winit/latest/winit/platform/run_return/trait.EventLoopExtRunReturn.html#tymethod.run_return
+    /// [`run_ondemand`]: https://docs.rs/winit/0.29.1-beta/winit/platform/run_ondemand/trait.EventLoopExtRunOnDemand.html#tymethod.run_ondemand
```

Comment:
wrong docs, I chose to go with pump_events ; not sure what's the best though, I gotta read https://github.com/rust-windowing/winit/issues/2900 more attentively.

Reply:
Fixed to `run_ondemand` ; but:

`run() -> !` has been changed to a `Result` https://github.com/rust-windowing/winit/pull/2767/files#diff-f5a20b394a5ba72e0e625bac291ac8e2fa04a59b2801dae39b277499d14729fdR315 ; I think it makes the example `return_after_run` (and its property `return_from_run`) useless, any feelings about that ?

Should I straight up remove that example and exposed `return_from_run` from `WinitSettings` ?